addons/website/tests/test_iap.py

```diff
@@ -5,7 +5,7 @@
 import odoo.tests
 from odoo.addons.iap.tools import iap_tools
 
-@odoo.tests.tagged('external')
+@odoo.tests.tagged('website_nightly', '-standard')
```

Comment:
@Xavier-Do 
Following [your comment](https://github.com/odoo/odoo/pull/123722#discussion_r1229209442), here it is. I suppose you will do the necessary to make it work correctly on the runbot :)

Let me know if I can do anything else to help :+1: 

Otherwise you can r+ this PR